### PR TITLE
MathUtil class

### PR DESCRIPTION
**Pull request by MariiaBorovykova**
review it

### CODE REVIEW

File: src/utils/MathUtils.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package utils;
+
+public class MathUtils {
+
```
Comment (merverdogan) [inline src/utils/MathUtils.java:4]: // make a change

**Review by merverdogan — COMMENTED** [review]
added new comment

**Comment by akbaha** [comment]
i make the review

**Comment by akbaha** [comment]
write and add comments

**Comment by akbaha** [comment]
one more comment

**Review by merverdogan — APPROVED** [review]

**Pull request merged**